docs/AboutUs.md

```diff
@@ -9,15 +9,15 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 
 ## Project team
 
-### John Doe
+### Samantha Low
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/samanthalow.png" width="200px">
```

Comment:
I think "the filename of the profile photo should be doc/images/githbub_username_in_lower_case.png" - Week 7 Point 2 in Project